modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -134,6 +158,82 @@ std::string HttpSynchronization::generateUid() {
     return fmt::format("{}/{}", _identifier, _version);
 }
 
+void HttpSynchronization::createSyncFile(bool fullySynchronized = true) const {
+    std::filesystem::path dir = directory();
+    std::filesystem::create_directories(dir);
+
+    dir.replace_extension("ossync");
+    std::ofstream syncFile(dir, std::ofstream::out);
+
+    syncFile << _ossyncVersionNumber << '\n' <<
+        (fullySynchronized ? _synchronizationToken : "Partial Synchronized") << '\n';
+
+    if (fullySynchronized)
+        return; //All files successfully downloaded, no need to write anything else to file.
```

Comment:
```suggestion
    if (fullySynchronized) {
        return; // All files successfully downloaded, no need to write anything else to file
    }
```

Even single instruction statements get `{` and `}`